include/net/download_client.h

```diff
@@ -60,20 +60,23 @@ enum download_client_evt_id {
 	DOWNLOAD_CLIENT_EVT_DONE,
 };
 
+struct download_fragment {
+    const void *buf;
```

Comment:
nit: Please indent with tabulation. Here and one line below.

Reply:
Ack, sorry, thought I had tabs for everything.